content/en/docs/collector/internal-telemetry.md

```diff
@@ -43,6 +43,21 @@ service:
                 port: 8888
 ```
 
+Alternatively, the Collector can push its internal metrics via to an OTLP
+backend via the following configuration:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      readers:
+        - periodic:
+            exporter:
+              otlp:
+                protocol: grpc/protobuf
+                endpoint: http://localhost:14317
+```
```

Comment:
I vote you put this first and save the `Alternatively` wording for the prometheus export.

Reply:
Done